buoi9/main.js

```diff
@@ -0,0 +1,93 @@
+//bai 1:nhap luong nhan vien,tinh thue thu nhap va luong rong
+const getsalary =(salary) =>{
+    if (salary >= 15000000) {
+        console.log('Thuế thu nhập', salary * 0.3)
+        console.log('Lương ròng   ', salary * 0.7)
+    } else if (salary >= 7000000) {
+        console.log('Thuế thu nhập', salary * 0.2)
+        console.log('Lương ròng   ', salary * 0.8)
+    } else {
+        console.log('Thuế thu nhập', salary * 0.1)
+        console.log('Lương ròng   ', salary * 0.9)
+    }
+}
+//bai 2:nhap tuoi va in ra dieu kien vao lop 10
+const checkAge =(age) =>{
+    if(age>=16)console.log('Du dieu kien vao lop 10')
+    else console.log('Khong du dieu kien vao lop 10')
+}
+//bai 3:nhap so nguyen bat ky in ra ket qua kiem tra so do lon hay nho hon 100
+const checkNumber=(number)=>{
+    if(number==100)console.log('So nay =100')
+    else if(number>100)console.log('So nay lon hon 100')
+    else  console.log('So nay nho hon 100')
+}
+//bai 4:Nhap vao 3 so nguyen va tim so lon nhat trong 3 so
+const checkMax = (firstNumber,secondNumber,thirdNumber)=>{
```

Comment:
```suggestion
const checkMaxNumber = (firstNumber, secondNumber, thirdNumber) => {
```
Nhìn kỹ các dấu cách, sửa lại toàn bộ các file